src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
+
+const clickPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject('Promise was rejected!');
+  }, 3000);
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+    clearTimeout(timeoutId);
+  });
+});
+
+const messageDiv = document.createElement('div');
+
+document.body.append(messageDiv);
```

Comment:
You are appending the messageDiv to the body before the promise resolves or rejects. This means that an empty div will be added to the body as soon as the script runs. It might be better to append the div to the body only after you know that you have a message to display.